base/ltsect.dtx

```diff
@@ -762,8 +762,8 @@
 % \end{macro}
 %
 %  \begin{macro}{\addcontentsline}
-%  The |\addcontentsline{|\meta{table}|}{|\meta{type}|}{|%
-%  \meta{entry}|}| command allows the user to  add
+%  The |\addcontentsline{|\meta{table}|}{|\meta{type}|}{|\meta{entry}|}|
+%  command allows the user to add
```

Comment:
To emit the surplus space character before `<entry>`:
![image](https://github.com/user-attachments/assets/730ba4a1-f984-480f-b747-98584a8982ec)